src/python/WMCore/ReqMgr/DataStructs/RequestStatus.py

```diff
@@ -77,6 +77,14 @@
 
 # if the state is not defined here (new, assignment-approved) allows all the property to get 
 ALLOWED_ACTIONS_FOR_STATUS = {
```

Comment:
without specifying anything mean it can add everything. If we set "new": ["RequestPriority"], means we can only set ["RequestPriority"] in new state. 
Which means we don't allow to create new request using web interface, Do you want to do that?